launchable/test_runners/perl.py

```diff
@@ -0,0 +1,58 @@
+import click
+from junitparser import TestCase, TestSuite  # type: ignore
+
+from ..testpath import TestPath
+from . import launchable
+
+
+@launchable.subset
+def subset(client):
```

Comment:
Just a Question: Why did you adopt this style and don't adopt the below style? 

```
subset = launchable.CommonSubsetImpls(__name__).scan_files('*.t')
```

Reply:
I see there are some Perl project using `.test` as extension to test file, while most of them use `.t`. I felt better to be able to support both.

Reply:
I see, thanks